### DISCUSSION TITLE
[docs] progress circular component not displaying the correct component

### DISCUSSION THREAD

**Discussion by tobi-oye** [chakra-ui/ark#2483] [General]
### Description

When I clicked on **progress circular** component in the docs, i expected to show the progress circular component but the **progress linear** component was displayed instead.

### Link to Reproduction (or Detailed Explanation)

https://ark-ui.com/docs/react/components/progress-circular

### Steps to Reproduce

1. Go to https://ark-ui.com/ 
2. Click on Docs 
3. Within the component section click on **Progress-Circular**

<img width="906" alt="image" src="https://github.com/chakra-ui/ark/assets/38209697/f89e7bbd-1c3d-4c9c-b2bc-d72a9b13d1ef">

### Ark UI Version

v3

### Framework

- [ ] React
- [ ] Solid
- [ ] Vue

### Browser

Chrome 124.0.6367.207 (Official Build) (arm64) 

### Additional Information

This library is really amazing and we appreciate all the hard work put into this 🎉🙏🙌🎉🙏🙌🎉🙏🙌

**Comment by cschroeter** [comment]
@tobi-oye 

Thanks - fixed.
